src/pages/Compare/index.spec.tsx

```diff
@@ -128,6 +128,23 @@ describe(__filename, () => {
     expect(root.find(VersionChooser)).toHaveProp('addonId', addonId);
   });
 
+  it('renders a loading message when no diff has been loaded', () => {
+    const addonId = 123;
+    const baseVersionId = 1;
+    const version = { ...fakeVersionWithDiff, id: baseVersionId + 1 };
+    const store = configureStore();
+    store.dispatch(versionsActions.loadVersionInfo({ version }));
+
+    const root = render({
+      addonId: String(addonId),
```

Comment:
I'm not sure why you're overriding `addonId` - it doesn't seem to have any impact on the test. Note that this is reflected in by diff above.